includes/admin/class-amp-post-meta-box.php

```diff
@@ -64,4 +91,69 @@ public function enqueue_admin_assets() {
 		) );
 	}
 
+	/**
+	 * Render AMP status.
+	 *
+	 * @since 0.6
+	 * @param object $post \WP_POST object.
+	 */
+	public function render_status( $post ) {
+		$verify = (
+			isset( $post->ID )
+			&&
+			isset( $post->post_type )
+			&&
+			post_type_supports( $post->post_type, AMP_QUERY_VAR )
+			&&
+			current_user_can( 'edit_post', $post->ID )
+		);
+
+		if ( true !== $verify ) {
+			return;
+		}
+
+		$status = get_post_meta( $post->ID, self::POST_META_KEY, true );
+		$labels = array(
+			'enabled'  => __( 'Enabled', 'amp' ),
+			'disabled' => __( 'Disabled', 'amp' ),
+		);
+
+		// Set default.
+		if ( empty( $status ) ) {
+			$status = 'enabled';
+		}
+
+		include_once AMP__DIR__ . '/templates/admin/amp-status.php';
+	}
+
+	/**
+	 * Save AMP Status.
+	 *
+	 * @since 0.6
+	 * @param int $post_id The Post ID.
+	 */
+	public function save_amp_status( $post_id ) {
+		$verify = (
+			isset( $_POST[ self::NONCE_NAME ] )
+			&&
+			isset( $_POST[ self::POST_META_KEY ] )
+			&&
+			wp_verify_nonce( sanitize_key( wp_unslash( $_POST[ self::NONCE_NAME ] ) ), self::NONCE_ACTION )
+			&&
+			current_user_can( 'edit_post', $post_id )
+			&&
+			! wp_is_post_revision( $post_id )
+			&&
+			! wp_is_post_autosave( $post_id )
+		);
+
+		if ( true === $verify ) {
+			update_post_meta(
+				$post_id,
+				self::POST_META_KEY,
+				sanitize_key( wp_unslash( $_POST[ self::POST_META_KEY ] ) )
```

Comment:
Note that this line will continue to be valid in WPCS once https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1222 is merged because `sanitize_key()` is marked in that PR as an `autoSlashingFunctions` (because it doesn't return any chars that will include slashes).

Reply:
Good to know 😄  Would you like to have it removed and add a whitelisting comment until it is merged?